src/components/Entry.vue

```diff
@@ -525,14 +525,13 @@ export default {
      */
     async authenticateID(userID) {
       this.isLoading = true;
-
       let userValidationResponse = await validateID(
         userID,
-        this.groupData.dataSource,
         this.authType,
         this.validateCount,
         this.dateOfBirth,
-        this.isExtraInputValidationRequired
+        this.isExtraInputValidationRequired,
+        this.$refs.phoneNumberEntry.phoneNumber
```

Comment:
Why is `phoneNumberEntry` coming from `$refs` unlike the others?

Reply:
Because `phoneNumberEntry` is a separate component. The ID and DOB are in the same `Entry` component. Moving forward I am creating a separate component for each kind of entry type, so that they can be mixed and matched for different forms.